src/db/postgres/lrsql/postgres/record.clj

```diff
@@ -54,6 +54,7 @@
     (alter-admin-account-passhash-optional! tx)
     (alter-admin-account-add-openid-issuer! tx)
     (alter-scope-enum-type! tx)
+    (alter-scope-enum-type-2! tx)
```

Comment:
As I said below, this should be it's own migration function, probably with a more descriptive name. I'd also wrap it with a query to see if it needs to be executed.

Reply:
Not sure what would be a more descriptive name. I guess the query wrapping would reduce overhead but I'm not sure if the overhead saved would be more than the extra overhead needed for the query.

Now that I'm looking at the function pair, it might be a good idea to _replace_ the original `alter-scope-enum-type!` with `alter-scope-enum-type-2!`, since the former's changes will get overwritten anyways.

Reply:
Did the replacement in ce69659e28bc5efad3c222d345ce50bda9e7a957 after performing the integration tests needed to see that the migration preserves existing scopes. Also renamed each function to `*-v2!` in 6287914af37b81f16d06c42bee2166fc182c88da to highlight the fact that the original migration functions are now superseded.